build.sbt

```diff
@@ -70,7 +70,22 @@ lazy val zioConfig =
     )
 
 lazy val examples = module("examples")
+  .settings(
+    skip in publish := true,
+    moduleName := "zio-config-examples",
+    libraryDependencies ++= Seq(
+      "com.propensive" %% "magnolia" % "0.12.0"
```

Comment:
I recommend 0.12.2, it was released just a few days ago and fixes some annoying cases where you get compiler errors a few thousand lines long 😄

Reply:
Cool. I wil upgrade.